src/view/com/util/List.tsx

```diff
@@ -68,6 +69,7 @@ function ListImpl<ItemT>(
       onBeginDragFromContext?.(e, ctx)
     },
     onEndDrag(e, ctx) {
+      runOnJS(updateActiveViewAsync)()
```

Comment:
swore i added this in another pr, but maybe it got closed or something

Reply:
How bizarre, maybe it got lost in a merge?